README.md

```diff
@@ -0,0 +1,57 @@
+# Restaurant Javascript Application
+
+## The application is created as a part requirement of the Microverse fullstack developer course.
+
+#### In this application, I show cased the use of webpack and JavaScript modules. I also show cased import and export of modules.
+
+### Built With
+
+- Javascript
+- Visual Studio Code
+- Webpack
+
+### System dependencies
+
+- web Browser
+
+### Get project up and running
+
+```
+git clone https://github.com/chrystalme/RestaurantApp.git
+```
+
+```
+cd RestaurantApp
+```
+
+```
+npm install
+```
+
+## Live Link
```

Comment:
[optional] Since this link is not available, It would be better if you remove this `Live Link` title